src/open_r1/rewards.py

```diff
@@ -367,6 +368,12 @@ def extract_code(completion: str, language: str = "python") -> str:
     return extracted_answer
 
 
+def binary_code_reward(completions, **kwargs) -> list[float]:
+    rewards = code_reward(completions, **kwargs)
+    BINARY_THRESHOLD = 0.99
```

Comment:
Should this be a function arg that the user can set?

Reply:
I am going to say no for now and we can add the feature if we need to toggle it.